app1-2.js

```diff
@@ -0,0 +1,9 @@
+var question=0;
+var answers=true;
+    while(true){
+    if(answers=(prompt("日本の首都は？")!== "東京")){
```

Comment:
比較を行う場合は `=` ではなく `===` を用います。

```suggestion
    if (answers === (prompt("日本の首都は？") !== "東京")) {
```